src/components/ToDoComponent/SettingGoals/CreateGoals.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
```

Comment:
The screen is overflowing in smaller device and I can't scroll to access the dropdown options. Perhaps we have to make it responsive or maybe use ScrollView?
Also, is the header above with the title CreatingGoals and left arrow part a part of the design?

Reply:
https://github.com/Real-Dev-Squad/mobile-app/assets/33225002/d9866c01-dbd0-4165-be0f-a752c80e22bd

Reply:
Also, apart from not being able to scroll, as you can see, the header is still there which is not part of the design

Reply:
Earlier I wrapped everything under used`safeAreaView` but now I have changed it to `scrollView`, you can find it in the latest commit.
Is this the APK which I have shared with you? If yes, after it, I have made many other changes, I removed the header and back arrow buttons as these both are not part of the design. In my last commit, you can see the changes.

Reply:
![photo_2023-06-12_11-33-37](https://github.com/Real-Dev-Squad/mobile-app/assets/57326919/e4c7ace2-66ab-45d8-9eae-9d595e35dc05)

https://github.com/Real-Dev-Squad/mobile-app/assets/57326919/bcf6515f-5f10-4839-883a-164be05d4e29

@sakshambhatt

Reply:
I have to test it on my phone to sign it off. Could you please provide an APK?

Reply:
@JAYATEJA04 please provide apk to @sakshambhatt

Reply:
I have checked, I am not facing any issue while scrolling the form , resolving the above conversation by this comment,